docs/contributing.rst

```diff
@@ -100,11 +119,22 @@ starting point is to send a pull request on GitHub. It can be simple enough to
 illustrate your idea, and leave documentation/tests for later, after the idea
 has been validated and proven useful. Alternatively, you can start a
 conversation in the `Scrapy subreddit`_ to discuss your idea first.
+
+Sometimes there is an existing pull request for the problem you'd like to
+solve, which is stalled for some reason. Often the pull request is in a
+right direction, but changes are requested by Scrapy maintainers, and the
+original pull request author haven't had time to address them.
+In this case consider picking up this pull request: open
+a new pull request with all commits from the original pull request, as well as
+additional changes to address the raised issues. Doing so helps a lot; it is
+not considered rude as soon as original the author is acknowledged by keeping
+his/her commits.
```

Comment:
+1, it is great to point that out explicitly!

It could be nice to add also that one can checkout a PR into a local branch by running:

    git fetch upstream pull/$PR_NUMBER/head:$BRANCH_NAME_TO_CREATE

Reply:
Awesome, I didn't know about this option!

Reply:
Yeah, it's pretty convenient, though a bit annoying to type all that each time. :)

I'm lazy so I setup [some aliases](https://github.com/eliasdorneles/dotfiles/blob/master/gitconfig#L81-L91) in order to just do `git pr $PR_NUMBER`